### DISCUSSION TITLE
2007/06/29/%E6%A8%A1%E5%9D%97%E6%8E%A5%E5%8F%A3-api-%E7%9A%84%E4%B8%A4%E7%A7%8D%E8%AE%BE%E8%AE%A1%E6%96%B9%E6%A1%88

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#171] [Announcements]
# 2007/06/29/%E6%A8%A1%E5%9D%97%E6%8E%A5%E5%8F%A3-api-%E7%9A%84%E4%B8%A4%E7%A7%8D%E8%AE%BE%E8%AE%A1%E6%96%B9%E6%A1%88

假如你要设计一个程序模块，它的功能是读写 INI 文件。用户调用这个模块，就可以方便的把信息写入 INI 文件，或从其中读出信息。

https://blog.qizhen.xyz/2007/06/29/%E6%A8%A1%E5%9D%97%E6%8E%A5%E5%8F%A3-api-%E7%9A%84%E4%B8%A4%E7%A7%8D%E8%AE%BE%E8%AE%A1%E6%96%B9%E6%A1%88

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2007年07月17日 09:23 编辑
回Mach，现在LabVIEW有个模块是 Internet Toolkit 可以解析 XML 文件。不过将来，LabVIEW 本身也会具备 XML 解析功能的。我自己设计程序中也是都开始使用 XML 文件来保存数据了。

回复
Qizhen
2007年07月5日 09:20 编辑
这个不是我，是他们弄错了。

回复
K
2007年07月5日 02:22 编辑
在NI的labview zone里看到你的推荐了。
Qizhen Ruan – LabVIEW Tips for Chinese Developers                       Software EngineerLabVIEW
不过不知道怎么连到一个英文的blog上了，也是你的吗？

回复
杰
2007年07月4日 23:12 编辑
还是觉得第二种比较好吧。
简单的程序用第一种，比较复杂的，或者继承性好的，还是第二种。
很明显的一点，如果采用第一种方法，有不少代码是重复的。
再深入到软件工程的角度的话，有点高深了，俺没有研究。呵呵。至少是没有把这样的编程经验提炼到广泛应用的角度。
